src/functions.sh

```diff
@@ -113,6 +113,31 @@ uploadSARIF () {
   fi
 }
 
+summary () {
+  local fixed_issues
+  fixed_issues=$(grep -Eo "[0-9]*" < <(csgrep --mode=stat ../fixes.log))
+
+  local added_issues
+  added_issues=$(grep -Eo "[0-9]*" < <(csgrep --mode=stat ../bugs.log))
+
+  echo -e "\
+### Differential ShellCheck 🐚
+
+Changed scripts: \`${#list_of_changed_scripts[@]}\`
+
+|                             | ❌ Added | ✅ Fixed |
+|:---------------------------:|:-------:|:-------:|
+| ⚠️ [Errors / Warnings / Notes](https://github.com/${GITHUB_REPOSITORY}/pull/${GITHUB_REF//merge}/files) |  **${added_issues:-0}**  |  **${fixed_issues:-0}**  |
```

Comment:
This link format does not seems to be working :/ See log of this PR: https://github.com/redhat-plumbers-in-action/differential-shellcheck/actions/runs/2591016887

It will point to [https://github.com/redhat-plumbers-in-action/differential-shellcheck/pull/ **refs/pull/41/** /files](https://github.com/redhat-plumbers-in-action/differential-shellcheck/pull/refs/pull/41//files) instead of https://github.com/redhat-plumbers-in-action/differential-shellcheck/pull/refs/pull/41/files.

Reply:
Thank you! I came across this issue but a forgot about it :smiley_cat:. This change should resolve the issue:

```
https://github.com/${GITHUB_REPOSITORY}/${${GITHUB_REF//refs\/}//\/merge}/files
```
link -> https://github.com/redhat-plumbers-in-action/differential-shellcheck/pull/41/files

I'll submit PR.